### ISSUE TITLE
Cast bar shield improvement

### ISSUE THREAD

**Issue by Romeoyoo** [mBlinkii/mMediaTag#99]
Hi!

First of all, a big thank you for the work. I really love the whole mMediaTag thing.

Secondly, I have a small request (or maybe it's already built-in, and I just don't know how to set it up. If so, then apologies) for a small improvement to the cast bar shield. When I set up a specific position for the Icon Anchor, it affects both individual frames and groups, such as bosses. It would be great to have the ability to set it up separately.

**Comment by mBlinkii** [comment]
Hello and thanks for your feedback, I didn't want to overload the small function with settings, so I added the auto position function. If you don't use this feature, the icon may not always be in the right place on the cast bar due to the different sizes.

**Comment by mBlinkii** [comment]
can you show me a screenshot where i can see what you want to achieve. i'm not going to add a separate anchor position for each unit, but maybe we can solve the problem differently.

**Comment by Romeoyoo** [comment]
![castbar shield](https://github.com/mBlinkii/mMediaTag/assets/113843561/06c14bb5-3620-4941-92b7-745f87456f58)
For the target, I couldn't achieve that by simply choosing the anchoring automatically. For the boss frame (I marked the spell icon with a red circle)shield icon will show up on the left while I want to have the spell icon on the right. I know I can have everything on the left side set up with the same width and it will look fine which i dont want to do. But in my opinion now it is not universal/ideal and causes a lot of errors, my idea would be similar. I got you I thought it ll be something simple, I’ll figure it out on my own, maybe with another addon or I’ll do something on my own in that case. Anyway, thank you very much and sorry for bothering.
